lms/static/styles/_utilities.scss

```diff
@@ -1,2 +1,49 @@
-@use '@hypothesis/frontend-shared/styles/util';
 @use 'tailwindcss/utilities';
+
+@layer utilities {
+  // Reusable patterns for possible extraction
+
+  // Apply basic styling to a list
+  .u-list {
+    @apply px-4 list-disc;
+  }
+
+  // Apply ordered-style to a list, with numbers
+  .u-list--decimal {
+    @apply u-list list-decimal;
+  }
+  .u-absolute-centered {
+    @apply absolute top-1/2 left-1/2 -translate-x-1/2 -translate-y-1/2;
+  }
+
+  // Copied current implementation of `hyp-u-outline-on-keyboard-focus`
+  // classes from the `frontend-shared` package.
+  // TODO: This should be implemented as a TW plugin in `frontend-shared`
+  // and replaced
+  .u-outline-on-keyboard-focus:focus {
```

Comment:
Looks like there is a lot of duplication here that could be eliminated. I assume you haven't done that because you're planning to replace it soon with the alternate method involving a TW plugin?

Reply:
Exactly. This is the inefficient c-n-p method that allows me to feel confident this definitely works as expected. If you didn't see it before, I pushed my working branch for an implementation swag at that: https://github.com/hypothesis/frontend-shared/compare/main...focus-visible-plugin